xonsh/platform.py

```diff
@@ -126,6 +126,26 @@ def is_readline_available():
 
 
 #
+# Dev release info
+#
+
+@functools.lru_cache(1)
+def githash():
+    from xonsh import main
+    install_base = main.__file__.rsplit('/', 1)[0]
```

Comment:
I knew there must be something to do that but didn't know what.  I learned a thing!